vaadin-text-field-flow-parent/vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/binder/BigDecimalFieldValidationTest.java

```diff
@@ -0,0 +1,47 @@
+package com.vaadin.flow.component.textfield.binder;
+
+import com.vaadin.flow.component.HasValue;
+import com.vaadin.flow.component.textfield.BigDecimalField;
+import com.vaadin.flow.function.SerializablePredicate;
+import org.junit.Ignore;
+
+import java.math.BigDecimal;
+
+public class BigDecimalFieldValidationTest
+        extends AbstractTextFieldValidationTest<BigDecimal> {
```

Comment:
How about we also add a generic type argument for the field's class? This way we could avoid duplicate private `field` properties and instead have one single protected property in the abstract class.

```suggestion
        extends AbstractTextFieldValidationTest<BigDecimal, BigDecimalField> {
```

Reply:
Makes sense. Refactored.